internal/blockchain/ethereum/ethsigner/ethsigner.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/hyperledger/firefly-cli/pkg/types"
 )
 
-var ethsignerImage = "ghcr.io/hyperledger/firefly-signer:v0.9.1"
+var ethsignerImage = "ghcr.io/hyperledger/firefly-signer:v0.9.6"
```

Comment:
Just noting that there is an ongoing conversation about what the "right" place for these is, and calling out that we're all in agreement it's not here. But it's here for now until we find a better spot for it.